### ISSUE TITLE
Unable to resolve path to module 'svelte-modals'.eslintimport/no-unresolved

### ISSUE THREAD

**Issue by dimahalaiskiy** [mattjennings/svelte-modals#34]
My eslint getting error: Unable to resolve path to module 'svelte-modals'.eslintimport/no-unresolved

The funny thing that it still works correctly

 "svelte": "^4.2.0",
 "svelte-modals": "^1.3.0",

**Comment by CogmedAppflow** [comment]
I also have issue with importing the module after upgrading to 1.3.0 but I get typescript error message. I temporarily solved it by adding a 'default' export to the package.json file, like this:
```
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "svelte": "./dist/index.js",
      "default": "./dist/index.js"
    }
  },
```

**Comment by mattjennings** [comment]
fixed in v2.0.0
